### PR TITLE
fix(nasha): add tracking for nasha

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nasha-refresh` 
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9031 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I added the tracking for nasha

### CODE REVIEW

File: packages/manager/modules/nasha/src/components/partition/access/delete/delete.controller.js [hunk 1]
```diff
@@ -16,4 +20,9 @@ export default class NashaComponentsPartitionAccessDeleteController {
       )
       .catch((error) => this.close({ error }));
   }
+
+  onClickOnCancel() {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/components/partition/access/delete/delete.controller.js:24]: **question:** There is a repetition in `onClickOnCancel`. Maybe `onCancelClick` sounds good.
```suggestion
  onCancelClick() {
```

File: packages/manager/modules/nasha/src/components/partition/create/create.controller.js [hunk 2]
```diff
@@ -46,4 +51,9 @@ export default class NashaComponentsPartitionCreateController {
       )
       .catch((error) => this.close({ error }));
   }
+
+  onClickOnClose() {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/components/partition/create/create.controller.js:55]: Same as https://github.com/ovh/manager/pull/7762/files#r943141172

File: packages/manager/modules/nasha/src/components/partition/delete/delete.controller.js [hunk 3]
```diff
@@ -15,4 +18,9 @@ export default class NashaComponentsPartitionDeleteController {
       )
       .catch((error) => this.close({ error }));
   }
+
+  onClickOnClose() {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/components/partition/delete/delete.controller.js:22]: Same as https://github.com/ovh/manager/pull/7762/files#r943141172

File: packages/manager/modules/nasha/src/components/partition/edit-size/edit-size.controller.js [hunk 4]
```diff
@@ -31,4 +36,9 @@ export default class NashaComponentsPartitionEditSizeController {
       .then((tasks) => this.close({ tasks, partitionName }))
       .catch((error) => this.close({ error }));
   }
+
+  onClickOnClose() {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/components/partition/edit-size/edit-size.controller.js:40]: Same as https://github.com/ovh/manager/pull/7762/files#r943141172

File: packages/manager/modules/nasha/src/components/partition/zfs-options/zfs-options.controller.js [hunk 5]
```diff
@@ -66,6 +68,11 @@ export default class NashaComponentsPartitionZfsOptionsController {
       .catch((error) => this.close({ error }));
   }
 
+  onClickOnClose() {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/components/partition/zfs-options/zfs-options.controller.js:71]: Same as https://github.com/ovh/manager/pull/7762/files#r943141172

File: packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.controller.js [hunk 6]
```diff
@@ -162,7 +168,14 @@ export default class NashaDashboardPartitionAccessesController {
     }
   }
 
+  onClickOnCancelAccessForm() {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.controller.js:171]:
```suggestion
  onCancelAccessFormClick() {
```

File: packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.controller.js [hunk 7]
```diff
@@ -180,4 +193,9 @@ export default class NashaDashboardPartitionAccessesController {
         this.alertError(error);
       });
   }
+
+  onClickOnGoToDelete(ip) {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.controller.js:197]:
```suggestion
  onDeleteClick(ip) {
```

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js [hunk 8]
```diff
@@ -155,6 +160,16 @@ export default class NashaDashboardPartitionSnapshotsController {
       });
   }
 
+  onClickOnCancelCustomSnapshot() {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js:163]:
```suggestion
  onCancelCustomSnapshotClick() {
```

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js [hunk 9]
```diff
@@ -155,6 +160,16 @@ export default class NashaDashboardPartitionSnapshotsController {
...
+    this.trackClick(PREFIX_TRACKING_SNAPSHOT_POLICY, 'cancel-create-snapshot');
+    return this.hideCustomSnapshotForm();
+  }
+
+  onClickOnGoToDelete(customSnapshot) {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js:168]:
```suggestion
  onDeleteClick(customSnapshot) {
```

File: packages/manager/modules/nasha/src/dashboard/partitions/partitions.controller.js [hunk 10]
```diff
@@ -50,4 +58,9 @@ export default class NashaDashboardPartitionsController {
       })
       .catch((error) => this.alertError({ error }));
   }
+
+  onClickOnGoToPartitionsCreate() {
```
Comment (anooparveti) [inline packages/manager/modules/nasha/src/dashboard/partitions/partitions.controller.js:62]:
```suggestion
  onPartitionsCreateClick() {
```

File: packages/manager/modules/nasha/src/components/partition/access/delete/delete.controller.js (hunk 1 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/components/partition/access/delete/delete.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/components/partition/create/create.controller.js (hunk 2 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/components/partition/create/create.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/components/partition/delete/delete.controller.js (hunk 3 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/components/partition/delete/delete.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/components/partition/edit-size/edit-size.controller.js (hunk 4 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/components/partition/edit-size/edit-size.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/components/partition/zfs-options/zfs-options.controller.js (hunk 5 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/components/partition/zfs-options/zfs-options.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.controller.js (hunk 6 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.controller.js (hunk 7 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js (hunk 8 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js (hunk 9 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.controller.js]: It's applied.

File: packages/manager/modules/nasha/src/dashboard/partitions/partitions.controller.js (hunk 10 above)
Comment (ghyenne) [reply packages/manager/modules/nasha/src/dashboard/partitions/partitions.controller.js:62]: It's applied.

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**